src/yio-plugin/integration_threadadapter.h

```diff
@@ -51,16 +51,16 @@ class IntegrationThreadAdapter : public Integration {
     Q_INVOKABLE void disconnect() override;
     void             enterStandby() override;
     void             leaveStandby() override;
-    QStringList      getAllAvailableEntities() override;
-    Q_INVOKABLE void sendCommand(const QString& type, const QString& entity_id, int command,
+    QVariantList     getAllAvailableEntities() override;
+    Q_INVOKABLE void sendCommand(const QString& type, const QString& entityId, int command,
```

Comment:
I don't think we need a Q_INVOKABLE here. It's an internal helper class.
In my opinion all Q_INVOKABLE should be slots including enterStandby and leaveStandby.